landbosse/excelio/XlsxGenerator.py

```diff
@@ -113,16 +113,35 @@ def tab_costs_by_module_type_operation(self, rows):
             sheet.
         """
         worksheet = self.workbook.add_worksheet('costs_by_module_type_operation')
-        for idx, col_name in enumerate(['project_id', 'module', 'operation_id', 'type_of_cost', 'total_or_turbine', 'cost']):
+        for idx, col_name in enumerate(['Project ID',
+                                        'Number of turbines',
+                                        'Turbine rating MW',
+                                        'Module',
+                                        'Operation ID',
+                                        'Type of cost',
+                                        # 'Raw cost is total or per turbine',
+                                        # 'Raw cost',
+                                        'Cost per turbine',
+                                        'Cost per project',
+                                        'USD/kW per project']):
             worksheet.write(0, idx, col_name, self.header_format)
         for row_idx, row in enumerate(rows):
             worksheet.write(row_idx + 1, 0, row['project_id'])
-            worksheet.write(row_idx + 1, 1, row['module'])
-            worksheet.write(row_idx + 1, 2, row['operation_id'])
-            worksheet.write(row_idx + 1, 3, row['type_of_cost'])
-            worksheet.write(row_idx + 1, 4, row['total_or_turbine'])
-            worksheet.write(row_idx + 1, 5, row['cost'], self.accounting_format)
+            worksheet.write(row_idx + 1, 1, row['num_turbines'])
+            worksheet.write(row_idx + 1, 2, row['turbine_rating_MW'])
+            worksheet.write(row_idx + 1, 3, row['module'])
+            worksheet.write(row_idx + 1, 4, row['operation_id'])
+            worksheet.write(row_idx + 1, 5, row['type_of_cost'])
+
+            # If these lines are uncommented then column ordering will need to be changed
```

Comment:
If we want to display the `raw_cost` in the output (see email) then we can uncomment these lines.